### ISSUE TITLE
Add ability to delete a single message

### ISSUE THREAD

**Issue by TheNain38** [qTox/qTox#2837]
By right clicking it, it would show an option to delete it

**Comment by zetok** [comment]
Just to make sure - feature request is about deleting message only on one side, where user clicked to delete it, right?

Is there some use case that you have in mind?

As for implementation - I think that it would be beneficial to allow deleting multiple messages when multiple are selected.
Additionally, when user clicks on the option, there should appear confirmation box, defaulting to `No`/`Cancel` button.

**Comment by zetok** [comment]
~related: #1306

**Comment by TheNain38** [comment]
Yes, just on my side, also I think it would be better UX to have a "Cancel" button appear when you delete messages

**Comment by ProMcTagonist** [comment]
User needs to know which message they're about to delete. Hilighting chosen message plus a regret timer would be cool
